Purchases/Networking/DNSChecker.swift

```diff
@@ -15,6 +15,8 @@ import Foundation
 
 enum DNSChecker {
 
+    static let blockedErrorKey: String = "RevenueCat API Blocked"
```

Comment:
I'm not convinced these belong here, but I didn't want to add them to the Error extension.

Reply:
I think it's fine 👍 Maybe name it `blockedErrorUserInfoKey` to make it more clear?
I don't think there's a convention, but I wouldn't use spaces, maybe something like `RCAPIBlocked`.

Reply:
Done!